src/ck-seat.c

```diff
@@ -1,6 +1,8 @@
 /* -*- Mode: C; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 8 -*-
  *
  * Copyright (C) 2006 William Jon McCann <mccann@jhu.edu>
+ * Copyright (C) 2023 Serenity Cybersecurity, LLC <license@futurecrew.ru>
+ *                    Author: Gleb Popob <arrowd@FreeBSD.org>
```

Comment:
I think this change is too trivial to be copyrightable.

Reply:
One-line changes may take hours of debugging. Anyways, it is a requirement from the company I'm working for, so I can't remove it, even if I wanted. Is this really a problem? We don't change the license itself, the software stays open source.

Reply:
Yes but the time it takes isn't really an argument.
Time to debug could be skill dependent.
Copyright implies some sort of creative content.
Otherwise most bugfix PR/MR in OSS projects would not be merged imho.

Reply:
I consulted with our lawyer and she said it is acceptable in case of this PR.